<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FarecMainWinClass</class>
 <widget class="QMainWindow" name="FarecMainWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaRec</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/face-smile.png</normaloff>:/icons/face-smile.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Obraz wejściowy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="InputImgLbl">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Podgląd</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="PviewImgLbl">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <widget class="QMenu" name="menuZapisz">
     <property name="title">
      <string>&amp;Zapisz</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
     </property>
     <addaction name="actionWej_cie"/>
     <addaction name="actionPodgl_d"/>
    </widget>
    <addaction name="actionOtw_rz"/>
    <addaction name="menuZapisz"/>
    <addaction name="separator"/>
    <addaction name="action_Wyj_cie"/>
   </widget>
   <widget class="QMenu" name="menuOpsy">
    <property name="title">
     <string>Opsy</string>
    </property>
    <widget class="QMenu" name="menuOczy">
     <property name="title">
      <string>Oczy</string>
     </property>
     <addaction name="actionOczyHough"/>
     <addaction name="actionVPF"/>
     <addaction name="actionOczyApprox"/>
    </widget>
    <widget class="QMenu" name="menuVPF">
     <property name="title">
      <string>VPF</string>
     </property>
     <addaction name="actionHori"/>
     <addaction name="actionVert"/>
    </widget>
    <addaction name="actionAutoprep"/>
    <addaction name="separator"/>
    <addaction name="actionNa_szaro"/>
    <addaction name="actionKontrast"/>
    <addaction name="separator"/>
    <addaction name="actionSobel"/>
    <addaction name="actionOtsu"/>
    <addaction name="actionGauss"/>
    <addaction name="actionMedianowy"/>
    <addaction name="actionHough"/>
    <addaction name="action_redni"/>
    <addaction name="menuVPF"/>
    <addaction name="separator"/>
    <addaction name="actionTwarz"/>
    <addaction name="menuOczy"/>
    <addaction name="actionNos"/>
    <addaction name="actionUsta"/>
   </widget>
   <widget class="QMenu" name="menuObrazy">
    <property name="title">
     <string>&amp;Obrazy</string>
    </property>
    <addaction name="actionU_yj_podgl_du"/>
   </widget>
   <widget class="QMenu" name="menuNarz_dzia">
    <property name="title">
     <string>&amp;Narzędzia</string>
    </property>
    <widget class="QMenu" name="menuGradient">
     <property name="title">
      <string>Gradient</string>
     </property>
     <addaction name="actionGradPvwWej_cie"/>
     <addaction name="actionGradPvwPodgl_d"/>
    </widget>
    <widget class="QMenu" name="menuKlasyfikacja">
     <property name="title">
      <string>Klasyfikacja</string>
     </property>
     <addaction name="actionPoka_odcinki"/>
    </widget>
    <addaction name="menuGradient"/>
    <addaction name="menuKlasyfikacja"/>
   </widget>
   <widget class="QMenu" name="menuTwarz">
    <property name="title">
     <string>Twarz</string>
    </property>
    <addaction name="actionDodaj"/>
    <addaction name="actionWyszukaj"/>
    <addaction name="actionW_a_ciel"/>
   </widget>
   <widget class="QMenu" name="menuBD">
    <property name="title">
     <string>BD</string>
    </property>
    <addaction name="actionDbPo_cz"/>
    <addaction name="actionDbRoz_cz"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuObrazy"/>
   <addaction name="menuOpsy"/>
   <addaction name="menuNarz_dzia"/>
   <addaction name="menuTwarz"/>
   <addaction name="menuBD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="ActionsToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="OpstoolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="DbToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOtw_rz">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Otwórz…</string>
   </property>
   <property name="statusTip">
    <string>Otwiera plik z obrazem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNa_szaro">
   <property name="text">
    <string>Na szarość</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionOtsu">
   <property name="text">
    <string>Otsu</string>
   </property>
  </action>
  <action name="actionU_yj_podgl_du">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/system-switch-user.png</normaloff>:/icons/system-switch-user.png</iconset>
   </property>
   <property name="text">
    <string>Użyj podglądu</string>
   </property>
  </action>
  <action name="actionWej_cie">
   <property name="text">
    <string>Wejście…</string>
   </property>
  </action>
  <action name="actionPodgl_d">
   <property name="text">
    <string>Podgląd…</string>
   </property>
  </action>
  <action name="action_Wyj_cie">
   <property name="text">
    <string>&amp;Wyjście</string>
   </property>
  </action>
  <action name="actionGauss">
   <property name="text">
    <string>Gauss</string>
   </property>
  </action>
  <action name="actionMedianowy">
   <property name="text">
    <string>Medianowy</string>
   </property>
  </action>
  <action name="actionGradPvwWej_cie">
   <property name="text">
    <string>Wejście…</string>
   </property>
  </action>
  <action name="actionGradPvwPodgl_d">
   <property name="text">
    <string>Podgląd…</string>
   </property>
  </action>
  <action name="actionHough">
   <property name="text">
    <string>Hough</string>
   </property>
  </action>
  <action name="action_redni">
   <property name="text">
    <string>Średni</string>
   </property>
  </action>
  <action name="actionTwarz">
   <property name="text">
    <string>Twarz</string>
   </property>
  </action>
  <action name="actionAutoprep">
   <property name="text">
    <string>Autoprep</string>
   </property>
  </action>
  <action name="actionOczyHough">
   <property name="text">
    <string>Hough</string>
   </property>
  </action>
  <action name="actionHori">
   <property name="text">
    <string>Hori</string>
   </property>
  </action>
  <action name="actionVert">
   <property name="text">
    <string>Vert</string>
   </property>
  </action>
  <action name="actionVPF">
   <property name="text">
    <string>VPF</string>
   </property>
  </action>
  <action name="actionOczyApprox">
   <property name="text">
    <string>Approx</string>
   </property>
  </action>
  <action name="actionNos">
   <property name="text">
    <string>Nos</string>
   </property>
  </action>
  <action name="actionKontrast">
   <property name="text">
    <string>Kontrast</string>
   </property>
  </action>
  <action name="actionUsta">
   <property name="text">
    <string>Usta</string>
   </property>
  </action>
  <action name="actionKlasyfikacja">
   <property name="text">
    <string>Klasyfikacja</string>
   </property>
  </action>
  <action name="actionDodaj">
   <property name="text">
    <string>Dodaj</string>
   </property>
  </action>
  <action name="actionWyszukaj">
   <property name="text">
    <string>Wyszukaj</string>
   </property>
  </action>
  <action name="actionPoka_odcinki">
   <property name="text">
    <string>Pokaż odcinki</string>
   </property>
  </action>
  <action name="actionDbPo_cz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/network-connect.png</normaloff>:/icons/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>Połącz</string>
   </property>
  </action>
  <action name="actionDbRoz_cz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/network-disconnect.png</normaloff>:/icons/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Rozłącz</string>
   </property>
  </action>
  <action name="actionW_a_ciel">
   <property name="text">
    <string>Właściel…</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
